.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
         commonjs: true,
         es6: true,
         node: true,
+        jest: true,
```

Comment:
Is this only needed in https://github.com/guardian/frontend/blob/master/static/src/javascripts/.eslintrc.js?